ogre2/src/Ogre2Camera.cc

```diff
@@ -76,7 +76,7 @@ void Ogre2Camera::SetHFOV(const math::Angle &_angle)
 {
   BaseCamera::SetHFOV(_angle);
   double angle = _angle.Radian();
-  double vfov = 2.0 * atan(tan(angle / 2.0) / this->aspect);
+  double vfov = 2.0 * atan(tan(angle / 2.0) / this->AspectRatio());
```

Comment:
If `this->aspect = this->AspectRatio();` is required in `OgreCamera::SetHFOV`, then we should add it to `Ogre2` camera's as well